.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: nomaddo/cross-rpi
+    steps:
+      - checkout
+      - run:
+          name: download header
+          command: git clone https://github.com/doe300/VC4CLStdLib.git VC4CLStdLib
+      - run:
+          name: get libVC4CC.so url
+          command: curl "https://circleci.com/api/v1.1/project/github/doe300/VC4C/latest/artifacts?circle-token=${CIRCLECI_API_TOKEN}=master&filter=successful" --output /tmp/dump
```

Comment:
On my local machine, I can download this file (and the artifact files) without any CircleCI API-key. Do you know if there is anything wrong with my CircleCI configuration or does it generally work without an API token?

Reply:
Mmm... same as my local machie. 
I am not sure why, but currently remove them.